ec2_locals.tf

```
@@ -7,6 +7,8 @@ locals {
     ) : (
       length(aws_route53_record.cyral-sidecar-dns-record) == 1 ? aws_route53_record.cyral-sidecar-dns-record[0].fqdn : aws_lb.cyral-lb.dns_name
     )
+  protocol = "${var.external_tls_type == "no-tls" ? "http" : "https"}"
+  curl = "${var.external_tls_type == "tls-skip-verify" ? "curl -k" : "curl"}"
```

I believe this notation throws a warning in newer Terraform versions when using ternary operators and string interpolation. See suggestion:

```suggestion
  protocol = var.external_tls_type == "no-tls" ? "http" : "https"
  curl = var.external_tls_type == "tls-skip-verify" ? "curl -k" : "curl"
```

---

Tested & confirmed to work